code/modules/security_levels/security levels.dm

```diff
@@ -146,6 +162,8 @@
 			return "gamma"
 		if(SEC_LEVEL_EPSILON)
 			return "epsilon"
+		if (SEC_LEVEL_GREY)
```

Comment:
wtf are you guys on about theres no extra line here????????????????????????????????????????????????
line 166 is clearly: return"grey" 
>:(
line 203 though is a different story...